<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqAppIcon64.png</iconset>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPipeline" >
    <property name="title" >
     <string>&amp;Pipeline</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFilters" >
    <property name="title" >
     <string>F&amp;ilters</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuServer" >
    <property name="title" >
     <string>&amp;Server</string>
    </property>
    <addaction name="actionServerConnect" />
    <addaction name="actionServerDisconnect" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecentFiles" >
     <property name="title" >
      <string>Recent Files</string>
     </property>
    </widget>
    <addaction name="actionFileOpen" />
    <addaction name="menuRecentFiles" />
    <addaction name="separator" />
    <addaction name="actionFileLoadServerState" />
    <addaction name="actionFileSaveServerState" />
    <addaction name="separator" />
    <addaction name="actionFileSaveData" />
    <addaction name="actionFileSaveScreenshot" />
    <addaction name="actionFileSaveAnimation" />
    <addaction name="separator" />
    <addaction name="actionServerConnect" />
    <addaction name="actionServerDisconnect" />
    <addaction name="separator" />
    <addaction name="actionFileExit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuAddPlot" >
     <property name="title" >
      <string>Add &amp;Plot</string>
     </property>
     <addaction name="actionHistogram" />
     <addaction name="actionXY_Plot" />
    </widget>
    <addaction name="actionEditUndo" />
    <addaction name="actionEditRedo" />
    <addaction name="separator" />
    <addaction name="actionEditCameraUndo" />
    <addaction name="actionEditCameraRedo" />
    <addaction name="separator" />
    <addaction name="menuAddPlot" />
    <addaction name="separator" />
    <addaction name="actionEditSettings" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpAbout" />
    <addaction name="actionHelpHelp" />
    <addaction name="separator" />
    <addaction name="actionHelpEnableTooltips" />
   </widget>
   <widget class="QMenu" name="menuSources" >
    <property name="title" >
     <string>S&amp;ources</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuToolbars" >
     <property name="title" >
      <string>Toolbars</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuPlots" >
     <property name="title" >
      <string>Plots</string>
     </property>
    </widget>
    <addaction name="actionResetCamera" />
    <addaction name="actionPositiveX" />
    <addaction name="actionNegativeX" />
    <addaction name="actionPositiveY" />
    <addaction name="actionNegativeY" />
    <addaction name="actionPositiveZ" />
    <addaction name="actionNegativeZ" />
    <addaction name="separator" />
    <addaction name="menuToolbars" />
    <addaction name="menuPlots" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionToolsCreateCustomFilter" />
    <addaction name="actionToolsManageCustomFilters" />
    <addaction name="separator" />
    <addaction name="actionToolsDumpWidgetNames" />
    <addaction name="actionToolsRecordTest" />
    <addaction name="actionToolsRecordTestScreenshot" />
    <addaction name="actionToolsPlayTest" />
    <addaction name="actionTesting_Window_Size" />
    <addaction name="separator" />
    <addaction name="actionToolsPythonShell" />
   </widget>
   <widget class="QMenu" name="menuAnimation" >
    <property name="title" >
     <string>&amp;Animation</string>
    </property>
    <addaction name="actionVCRFirstFrame" />
    <addaction name="actionVCRPreviousFrame" />
    <addaction name="actionVCRPlay" />
    <addaction name="actionVCRPause" />
    <addaction name="actionVCRNextFrame" />
    <addaction name="actionVCRLastFrame" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuView" />
   <addaction name="menuSources" />
   <addaction name="menuFilters" />
   <addaction name="menuAnimation" />
   <addaction name="menuTools" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>671</y>
     <width>920</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="pipelineBrowserDock" >
   <property name="windowTitle" >
    <string>Pipeline Browser</string>
   </property>
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2" >
    <property name="minimumSize" >
     <size>
      <width>100</width>
      <height>16</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="objectInspectorDock" >
   <property name="windowTitle" >
    <string>Object Inspector</string>
   </property>
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3" />
  </widget>
  <widget class="QDockWidget" name="statisticsViewDock" >
   <property name="windowTitle" >
    <string>Statistics View</string>
   </property>
   <property name="allowedAreas" >
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4" />
  </widget>
  <widget class="QDockWidget" name="elementInspectorDock" >
   <property name="windowTitle" >
    <string>Element Inspector</string>
   </property>
   <property name="allowedAreas" >
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5" />
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <property name="windowTitle" >
    <string>Main Controls</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>20</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionFileOpen" />
   <addaction name="actionFileSaveData" />
   <addaction name="separator" />
   <addaction name="actionServerConnect" />
   <addaction name="actionServerDisconnect" />
   <addaction name="separator" />
   <addaction name="actionHelpHelp" />
  </widget>
  <widget class="QToolBar" name="selectionToolbar" >
   <property name="windowTitle" >
    <string>Selection Controls</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>20</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionMoveMode" />
   <addaction name="actionSelectionMode" />
  </widget>
  <widget class="QToolBar" name="variableToolbar" >
   <property name="windowTitle" >
    <string>Active Variable Controls</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>20</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionScalarBarVisibility" />
  </widget>
  <widget class="QToolBar" name="representationToolbar" >
   <property name="windowTitle" >
    <string>Representation</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>20</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
  <widget class="QToolBar" name="customFilterToolbar" >
   <property name="windowTitle" >
    <string>Custom Filter Controls</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>20</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
  <widget class="QToolBar" name="undoRedoToolbar" >
   <property name="windowTitle" >
    <string>Undo/Redo Controls</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>20</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionEditUndo" />
   <addaction name="actionEditRedo" />
  </widget>
  <widget class="QToolBar" name="VCRToolbar" >
   <property name="windowTitle" >
    <string>VCR Controls</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>20</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionVCRFirstFrame" />
   <addaction name="actionVCRPreviousFrame" />
   <addaction name="actionVCRPlay" />
   <addaction name="actionVCRPause" />
   <addaction name="actionVCRNextFrame" />
   <addaction name="actionVCRLastFrame" />
  </widget>
  <widget class="QToolBar" name="cameraToolbar" >
   <property name="windowTitle" >
    <string>Camera Controls</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>20</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionResetCamera" />
   <addaction name="actionPositiveX" />
   <addaction name="actionNegativeX" />
   <addaction name="actionPositiveY" />
   <addaction name="actionNegativeY" />
   <addaction name="actionPositiveZ" />
   <addaction name="actionNegativeZ" />
  </widget>
  <action name="actionFileOpen" >
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqOpen22.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileExit" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelpAbout" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionHelpHelp" >
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqHelp22.png</iconset>
   </property>
   <property name="text" >
    <string>Help</string>
   </property>
  </action>
  <action name="actionScalarBarVisibility" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>SbV</string>
   </property>
  </action>
  <action name="actionServerConnect" >
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqConnect22.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionServerDisconnect" >
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqDisconnect22.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionFileSaveScreenshot" >
   <property name="text" >
    <string>Save Screenshot</string>
   </property>
  </action>
  <action name="actionFileSaveAnimation" >
   <property name="text" >
    <string>Save Animation</string>
   </property>
  </action>
  <action name="actionVCRPreviousFrame" >
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqVcrBack22.png</iconset>
   </property>
   <property name="text" >
    <string>Pre&amp;vious Frame</string>
   </property>
  </action>
  <action name="actionVCRFirstFrame" >
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqVcrFirst22.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;First Frame</string>
   </property>
  </action>
  <action name="actionVCRNextFrame" >
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqVcrForward22.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Next Frame</string>
   </property>
  </action>
  <action name="actionVCRLastFrame" >
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqVcrLast22.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Last Frame</string>
   </property>
  </action>
  <action name="actionFileSaveData" >
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqFloppyDisk22.png</iconset>
   </property>
   <property name="text" >
    <string>Save Data</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileLoadServerState" >
   <property name="text" >
    <string>&amp;Load Server State</string>
   </property>
  </action>
  <action name="actionFileSaveServerState" >
   <property name="text" >
    <string>&amp;Save Server State</string>
   </property>
  </action>
  <action name="actionHelpEnableTooltips" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Enable Tooltips</string>
   </property>
  </action>
  <action name="actionMoveMode" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqPointer22.png</iconset>
   </property>
   <property name="text" >
    <string>Move Mode</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="actionSelectionMode" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqSelectPointer22.png</iconset>
   </property>
   <property name="text" >
    <string>Selection Mode (s)</string>
   </property>
   <property name="shortcut" >
     <string/>
   </property>
  </action>
  <action name="actionEditUndo" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqUndo22.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionEditRedo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqRedo22.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEditCameraUndo" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Camera Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionEditCameraRedo" >
   <property name="icon" >
    <iconset/>
   </property>
   <property name="text" >
    <string>Camera Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionToolsDumpWidgetNames" >
   <property name="text" >
    <string>&amp;Dump Widget Names</string>
   </property>
  </action>
  <action name="actionToolsRecordTest" >
   <property name="text" >
    <string>&amp;Record Test</string>
   </property>
  </action>
  <action name="actionToolsRecordTestScreenshot" >
   <property name="text" >
    <string>Record &amp;Test Screenshot</string>
   </property>
  </action>
  <action name="actionToolsPythonShell" >
   <property name="text" >
    <string>Python &amp;Shell</string>
   </property>
  </action>
  <action name="actionToolsCreateCustomFilter" >
   <property name="text" >
    <string>&amp;Create Custom Filter ...</string>
   </property>
  </action>
  <action name="actionToolsManageCustomFilters" >
   <property name="text" >
    <string>&amp;Manage Custom Filters ...</string>
   </property>
  </action>
  <action name="actionVCRPlay" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqVcrPlay22.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Play</string>
   </property>
  </action>
  <action name="actionVCRPause" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../../Qt/Components/Resources/pqResources.qrc" >:/pqWidgets/Icons/pqVcrPause22.png</iconset>
   </property>
   <property name="text" >
    <string>Pa&amp;use</string>
   </property>
  </action>
  <action name="actionToolsPlayTest" >
   <property name="text" >
    <string>Play Test</string>
   </property>
  </action>
  <action name="actionEditSettings" >
   <property name="text" >
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionResetCamera" >
   <property name="text" >
    <string>R</string>
   </property>
   <property name="statusTip" >
    <string>Reset Camera</string>
   </property>
   <property name="whatsThis" >
    <string>Reset Camera</string>
   </property>
  </action>
  <action name="actionPositiveX" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>+X</string>
   </property>
   <property name="toolTip" >
    <string>Set view direction to +X</string>
   </property>
   <property name="statusTip" >
    <string>Set view direction to +X</string>
   </property>
   <property name="whatsThis" >
    <string>Set view direction to +X</string>
   </property>
  </action>
  <action name="actionNegativeX" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>-X</string>
   </property>
   <property name="toolTip" >
    <string>Set view direction to -X</string>
   </property>
   <property name="statusTip" >
    <string>Set view direction to -X</string>
   </property>
   <property name="whatsThis" >
    <string>Set view direction to -X</string>
   </property>
  </action>
  <action name="actionPositiveY" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>+Y</string>
   </property>
   <property name="toolTip" >
    <string>Set view direction to +Y</string>
   </property>
   <property name="statusTip" >
    <string>Set view direction to +Y</string>
   </property>
   <property name="whatsThis" >
    <string>Set view direction to +Y</string>
   </property>
  </action>
  <action name="actionNegativeY" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>-Y</string>
   </property>
   <property name="toolTip" >
    <string>Set view direction to -Y</string>
   </property>
   <property name="statusTip" >
    <string>Set view direction to -Y</string>
   </property>
   <property name="whatsThis" >
    <string>Set view direction to -Y</string>
   </property>
  </action>
  <action name="actionPositiveZ" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>+Z</string>
   </property>
   <property name="toolTip" >
    <string>Set view direction to +Z</string>
   </property>
   <property name="statusTip" >
    <string>Set view direction to +Z</string>
   </property>
   <property name="whatsThis" >
    <string>Set view direction to +Z</string>
   </property>
  </action>
  <action name="actionNegativeZ" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>-Z</string>
   </property>
   <property name="toolTip" >
    <string>Set view direction to -Z</string>
   </property>
   <property name="statusTip" >
    <string>Set view direction to -Z</string>
   </property>
   <property name="whatsThis" >
    <string>Set view direction to -Z</string>
   </property>
  </action>
  <action name="actionHistogram" >
   <property name="text" >
    <string>&amp;Histogram</string>
   </property>
  </action>
  <action name="actionXY_Plot" >
   <property name="text" >
    <string>XY &amp;Plot</string>
   </property>
  </action>
  <action name="actionTesting_Window_Size" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Max Window Size: 300x300</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../../Qt/Components/Resources/pqResources.qrc" />
 </resources>
 <connections/>
</ui>
